R/registry.R

```diff
@@ -189,12 +191,53 @@ list_registered_models <- function(max_results = 100, page_token = NULL, client
     )
   )
 
-  return(
-    list(
-      registered_models = response$registered_model,
-      next_page_token = response$next_page_token
-    )
+  res <- list(
+    registered_models = response$registered_model,
+    next_page_token = response$next_page_token
+  )
+  if(isFALSE(parse)) {
+    return(res)
+  }
+  res$registered_models <- parse_registered_models(res$registered_models)
+  res
+}
+
+#' @importFrom purrr transpose
+parse_registered_models <- function(registered_models) {
```

Comment:
not sure if it's worth exporting. i think we should if we don't end up going with the `parse` param